### DISCUSSION TITLE
How to configure a project for both tauri and axum (web) targets?

### DISCUSSION THREAD

**Discussion by shi-yan** [tauri-apps/tauri#11399] [Q&A]
My app has both a web based interface and a tauri based app. 90% of the code should be the same. The web version has a backend either based on axum or AWS lambda sdk, how can I configure the code project to build for both targets from a single code base?

any way to check if the build target is tauri or not? how about the frontend side?

**Comment by mcitem** [comment]
You might be looking for
 [multi-environment builds](https://rsbuild.dev/guide/advanced/environments) 
[vite-plugin-conditional-compile](https://github.com/LZS911/vite-plugin-conditional-compile)
